tm2/pkg/bft/consensus/common_test.go

```diff
@@ -762,3 +763,48 @@ func newPersistentKVStore() abci.Application {
 func newPersistentKVStoreWithPath(dbDir string) abci.Application {
 	return kvstore.NewPersistentKVStoreApplication(dbDir)
 }
+
+// ------------------------------------
+
+func ensureDrainedChannels(t *testing.T, channels ...any) {
```

Comment:
I hope I can men-in-black my brain after seeing this reflect trickery for channels

Reply:
ahaha, I created this method as a general-purpose tool for debugging undrained channels to check for events that haven't been consumed yet in case of failure. It wasn't meant to be pushed initially, as it was a debug method. However, I discussed this with Jae, and we want to keep it in case of another CI failure so we have a trace for this.